paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodForm.kt

```diff
@@ -60,7 +61,7 @@ internal fun PaymentMethodForm(
     modifier: Modifier = Modifier,
 ) {
     LaunchedEffect(paymentMethodCode) {
-        completeFormValues.collect {
+        completeFormValues.distinctUntilChanged().collect {
```

Comment:
Does this help reduce the number of callbacks sent to `onFormFieldValuesChanged`?

Reply:
Yup! I have a test that was annoying to consume all these events, so I just added a filter here.